packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -108,6 +109,13 @@ interface MongoNavInterface {
    * Applies a className to the root element
    */
   className?: string;
+
+  /**
+   * Click EventHandler that receives a `type` as its first argument and the associated `MouseEvent` as its second
+   * `type` can be ['logout', 'cloud', 'realm', 'charts']
```

Comment:
I wouldn't include this here, just ask the user to refer to the object, so there's no risk in things getting out of sync.

Reply:
👍